<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThinFilmWidget</class>
 <widget class="QWidget" name="ThinFilmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="thickness_spinbox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="prefix">
    <string>Thickness: </string>
   </property>
   <property name="suffix">
    <string> nm</string>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000100000000000</double>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="conductor_checkbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>87</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>conductor</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="refractive_index_frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="extinction_frame">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
